packages/expo-notifications/android/src/main/java/expo/modules/notifications/notifications/background/BackgroundRemoteNotificationTaskConsumer.java

```diff
@@ -7,6 +7,7 @@
 import android.os.PersistableBundle;
 import android.util.Log;
 
+import org.jetbrains.annotations.NotNull;
```

Comment:
Same goes for this one:

```suggestion
import androidx.annotation.NonNull;
`

Reply:
I used wrong annotation - NotNull instead of NonNull, hence the wrong import. Thanks!